Assets/UnityTestTools/NativeTesting/iOS/ApplePayEventTester.cs

```diff
@@ -1,5 +1,4 @@
-﻿#if UNITY_IPHONE 
```

Comment:
@dengjeffrey Seems that with this #ifdef check the tests won't work but without it works.